docs/specification/Feb2020Ballot.md

```diff
@@ -0,0 +1,534 @@
+
+Although FHIRcast works best with the SMART on FHIR launch and authorization process, implementation-specific launch, authentication, and authorization protocols may be possible. If not using SMART on FHIR, the mechanism enabling the app to discover the `hub.url` and `hub.topic` is not defined in FHIRcast. See [other launch scenarios](/launch-scenarios) for guidance.
+
+### FHIRcast Authorization & SMART scopes
+
+FHIRcast defines OAuth 2.0 access scopes that correspond directly to [FHIRcast events](#events). Our scopes associate read or write permissions to an event. Apps that need to receive workflow related events should ask for read scopes. Apps that request context changes should ask for write scopes. Hubs may decide what specific interactions and operations will be enabled by these scopes.
+
+Expressed in EBNF notation, FHIRcast's scope syntax is:
+
+`scope ::= ( 'fhircast' ) '/' ( FHIRcast-event ) '.' ( 'read' | 'write' | '*' )`
+
+![FHIRcast SMART scopes](../img/fhircast-smart-scopes.png)
+
+The FHIRcast event name is also a [computable syntax](#event-definition-format-hook-name), the complete syntax for FHIRcast scopes is:
+
+` scope    ::= 'fhircast' '/' fhir-resource '-' ( 'open' | 'close' | '*' ) '.' ( 'read' | 'write' | '*' )`
+
+![FHIRcast SMART scopes and event syntax](../img/fhircast-event-and-smart-scope.png)
+
+### SMART Launch Example
+Note that the SMART launch parameters include the Hub's base url and the session identifier in the `hub.url` and `hub.topic` fields.
+
+```
+{
+  "access_token": "i8hweunweunweofiwweoijewiwe",
+  "token_type": "bearer",
+  "patient":  "123",
+  "expires_in": 3600,
+  "encounter": "456",
+  "imagingstudy": "789",
+  "hub.url" : "https://hub.example.com",
+  "hub.topic": "fdb2f928-5546-4f52-87a0-0648e9ded065",
+}
+```
+
+
+## Subscribing and Unsubscribing
+
+Subscribing consists of two exchanges:
+
+* Subscriber requests a subscription at the `hub.url` url.
+* Hub confirms the subscription was actually requested by the subscriber by contacting the `hub.callback` url. 
+
+Unsubscribing works in the same way, except with a single parameter changed to indicate the desire to unsubscribe.
+
+### Subscription Request
+To create a subscription, the subscribing app SHALL perform an HTTP POST ([RFC7231](https://www.w3.org/TR/websub/#bib-RFC7231)) to the Hub's base url (as specified in `hub.url`) with the parameters in the table below.
+
+This request SHALL have a `Content-Type` header of `application/x-www-form-urlencoded` and SHALL use the following parameters in its body, formatted accordingly:
+
+Field | Optionality | Type | Description
+---------- | ----- | -------- | --------------
+`hub.channel.type` | Required | *string* | The subscriber SHALL specify a channel type of `websocket` | `webhook`. Subscription requests without this field SHOULD be rejected by the Hub.
```

Comment:
The pipe between `websocket` and `webhook` makes it not show up when actually viewing the file.